tests_metricflow/snapshots/test_predicate_pushdown_rendering.py/SqlQueryPlan/DuckDB/test_cumulative_metric_with_query_time_filters__plan0_optimized.sql

```diff
@@ -0,0 +1,49 @@
+-- Join Standard Outputs
+-- Pass Only Elements: ['bookers', 'listing__country_latest', 'booking__is_instant', 'metric_time__day']
+-- Pass Only Elements: ['bookers', 'listing__country_latest', 'metric_time__day']
+-- Aggregate Measures
+-- Compute Metrics via Expressions
+SELECT
+  subq_20.metric_time__day AS metric_time__day
+  , listings_latest_src_28000.country AS listing__country_latest
+  , COUNT(DISTINCT subq_20.bookers) AS every_two_days_bookers
+FROM (
+  -- Join Self Over Time Range
+  -- Pass Only Elements: ['bookers', 'booking__is_instant', 'metric_time__day', 'listing']
+  SELECT
+    subq_18.ds AS metric_time__day
+    , subq_16.listing AS listing
+    , subq_16.bookers AS bookers
+  FROM ***************************.mf_time_spine subq_18
+  INNER JOIN (
+    -- Constrain Output with WHERE
+    SELECT
```

Comment:
Would love if this subquery was combined with the inner subquery... but alas, a problem for a future date. Do we have a task up to fix that ?